contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -429,8 +429,9 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
          * liquidity in the pool yet. The full liquidity removal is thus skipped.
```

Comment:
nit: comment about "full liquidity being removed" is misleading

Reply:
thanks fixed